### PR TITLE
docs: populate geovista 101

### PR DESCRIPTION
**Pull request by bjlittle**
### Description

---

### CODE REVIEW

File: docs/src/quick_start.rst [hunk 1]
```diff
@@ -1,6 +1,342 @@
...
+:term:`land mask <Land Mask>` cells, revealing a `NASA Blue Marble`_
+:term:`textured mapped <Texture Map>` :term:`base layer <Base Layer>`
+underneath.
+
+*Sea Surface Temperature* data located on the grid cells are rendered using
+the :term:`perceptually uniform <Perceptually Uniform Colormap>`
+`cmocean balance`_ diverging colormap, along with
+`10m Natural Earth coastlines`_.
+
+The *grid*, *coastlines* and *base layer* are then transformed to the
```
Comment (pp-mo) [inline docs/src/quick_start.rst:90]: There is no grid, if that means a graticule ??

**Review by pp-mo — APPROVED** [review]
This is a huge leap forward.
I've suggested a few things, but I'm not blocking you with "request changes"  as I won't be around to discuss tomorrow !

File: docs/src/quick_start.rst (hunk 1 above)
Comment (bjlittle) [reply docs/src/quick_start.rst:90]: There is no graticule in this example, and I can see the confusion, so let's avoid that. I'll simply use the term `mesh`.

I was hanging on to the terminology of rectilinear and curvilinear being grids, and unstructured mesh ... although through `pyvista` we create a mesh in all cases.

File: docs/src/quick_start.rst [hunk 2]
```diff
@@ -1,6 +1,342 @@
...
+  The **Interactive Scene**
+  `vtk.js <https://kitware.github.io/vtk-js/index.html>`_ backend does **not**
+  support rendering text or points as spheres.
+
+Let's explore various atmospheric and oceanographic model data using
+``geovista``, which makes it easy to visualize **rectilinear**,
+**curvilinear**, and **unstructured** meshes.
+
+
+OISST AVHRR
```
[the remark this answers is not in the document]
Comment (bjlittle) [reply docs/src/quick_start.rst:69]: I'd like to keep that data sources, as that matches the `README.md`, and Earth Scientists will hone in on this, but the explanation in the text will clarify for those unfamiliar.

Also, yeah, I'll simplify the first example, remove the threshold and base layer, and also the projection 👍 

But I'll still keep the order of examples as-is. I think it's fine. Plus the examples are building up from rectilinear, curvilinear then on to unstructured.

**Pull request merged**